basement_bot/admin.py

```diff
@@ -304,3 +305,68 @@ async def _bot_data(self, ctx):
         embed.set_thumbnail(url=self.bot.user.avatar_url)
 
         await self.bot.tagged_response(ctx, embed=embed)
+
+    @decorate.with_typing
+    @commands.command(name="issue", aliases=["ish", "botish", "botissue"])
+    async def issue(self, ctx, title: str, description: str):
+        """Creates an issue in the bot's Github Repo
+
+        This is a command and should be accessed via Discord.
+
+        parameters:
+            ctx (discord.ext.Context): the context object for the calling message
+            title: the title of the issue
+            description: the description of the issue
+        """
+        GITHUB_API_BASE_URL = "https://api.github.com"
+        ICON_URL = (
+            "https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png"
+        )
+
+        oauth_token = self.bot.config.main.api_keys.github
+        if not oauth_token:
+            await self.bot.tagged_response(
+                ctx, "I couldn't authenticate with Github", target=ctx.author
+            )
+            return
+
+        headers = {
+            "Authorization": f"Bearer {oauth_token}",
+            "Accept": "application/vnd.github.v3+json",
+            "Content-Type": "text/plain",
+        }
+
+        data = {"title": f"{title}", "body": f"{description}"}
```

Comment:
You don't need f-strings since it's just the variable

(python for vscode should point this out too)